docs/twig-components/filters.md

```diff
@@ -201,6 +202,20 @@ or:
 and this case sensitivity is undesirable, you can use `|order('slug')` instead.
 The slug is always lowercase, so this will normalize the ordering.
 
+### ordering using Twig's <code>sort</code> filter.
+
+You can also order records using Twig's default `sort` filter, which may give you
+finer control over the ordering. For example, you can order the related pages by
+their group like so:
+
+```twig
+<ul>
+{% for page in record|related|sort((a, b) => a|taxonomies['groups'] <=> b|taxonomies['groups']) %}
+    <li><a href="{{ page|link }}">{{ page.title }}</a></li>
+{% endfor %}
+</ul>
+```
+
```

Comment:
Should we add here that the Spaceship operator only sorts ascending, and that if people want to flip it, they can do something like this? 

```twig
{% set pages = record|related|sort((a, b) => a|title <=> b|title) %}

<ul>
{% for page in pages|reverse %}    
    <li><a href="{{ page|link }}">{{ page.title }}</a></li>
{% endfor %}
</ul>
```